lib/ufe/UsdStageMap.h

```diff
@@ -22,22 +22,37 @@
 #include <pxr/usd/usd/stage.h>
 #include <pxr/base/tf/hash.h>
 
+#include <maya/MObjectHandle.h>
+
 #include <unordered_map>
 
+// Allow for use of MObjectHandle with std::unordered_map.
+namespace std {
+template <> struct hash<MObjectHandle> {
+    std::size_t operator()(const MObjectHandle& handle) const {
+      return static_cast<std::size_t>(handle.hashCode());
+    }
+};
+}
```

Comment:
This would be a great candidate for the kind of Maya-as-the-only-dependency utilities library that Animal Logic has been advocating for. We actually had something for this in usdMaya (now migrated to mayaUsd) as well:
https://github.com/Autodesk/maya-usd/blob/31e6f34afd6df370eed6bf4daab77ad420b98c54/lib/utils/util.h#L105

There's nothing USD or UFE related about either implementation.